bin/millau-runtime/src/lib.rs

```diff
@@ -316,7 +318,8 @@ construct_runtime!(
 		NodeBlock = opaque::Block,
 		UncheckedExtrinsic = UncheckedExtrinsic
 	{
-		BridgeCallDispatch: pallet_bridge_call_dispatch::{Module, Event<T>},
+		// TODO: Add Config for BridgeSubstrate
```

Comment:
I'm going to add this once we split the Rialto and Millau nodes since this gets added in the `chain_spec.rs` file and I don't want to feature flag it atm.